docs/static/jvm.asciidoc

```diff
@@ -5,7 +5,7 @@
 {ls} requires one of these versions:
 
 * Java 11
-* Java 17 (see <<jdk17-upgrade>> for settings info)
+* Java 17 (default, see <<jdk17-upgrade>> for settings info)
```

Comment:
@robbavey do we still need to refer customers to the settings who are installing or upgrading to the latest version of Logstash?

Reply:
Yes, I think so. There may still be users upgrading from Logstash <7.12 who may have updated their `config/jvm.options` file.